docs/index.rst

```diff
@@ -1066,7 +1066,18 @@ Here are some suggestions on how to troubleshoot the demo.
 
           cd scripts/validate/
 
+#. If running cluster for a while, and ``tools`` service returns:
 
+   .. sourcecode:: bash
+
+          Your token has expired. You are now logged out.
+          Error: You must log in to run that command.
+
+   Then run the following command to refresh the token:
+
+   .. sourcecode:: bash
+
+          ./scripts/validate/validate_tools_token.sh 
```

Comment:
@jeqo great idea to call this out in docs.

Feedback: 

First, consider a script name and location that reflect what it does, along those lines:

- instead of `scripts/validate/` (it does not validate), it could reside in `scripts/helper/`
- instead of `validate_tools_token.sh` (this isn't a "tools" token), call it `refresh_mds_login.sh`

Second, because this script will now exist, it should be referenced wherever else `mds_login` is called from the host:

```
[12:41:13] ~/git/cp-demo(5.5.x): grep -ir mds_login * | grep docker-compose
scripts/validate/validate_rest_proxy.sh:docker-compose exec tools bash -c ". /tmp/helper/functions.sh ; mds_login $MDS_URL ${SUPER_USER} ${SUPER_USER_PASSWORD}"
scripts/validate/validate_bindings.sh:docker-compose exec tools bash -c ". /tmp/helper/functions.sh ; mds_login $MDS_URL ${SUPER_USER} ${SUPER_USER_PASSWORD}"
```

Reply:
Done :) Though, scripts/validate/validate_bindings.sh doesn't have that line in branch `5.4.1-post`